pkg/manifests/manifests.go

```diff
@@ -1867,6 +1873,9 @@ func (f *Factory) PrometheusOperatorDeployment(namespaces []string) (*appsv1.Dep
 			args[i] = PrometheusOperatorPrometheusInstanceNamespacesFlag + f.namespace
 		}
 	}
+	if logLevel != "" {
+		args = append(args, fmt.Sprintf("--log-level=%s", logLevel))
+	}
```

Comment:
(nit) we could get rid of the `logLevel` variable?

```suggestion
	if f.config.ClusterMonitoringConfiguration.PrometheusOperatorConfig.LogLevel != "" {
		args = append(args, fmt.Sprintf("--log-level=%s", f.config.ClusterMonitoringConfiguration.PrometheusOperatorConfig.LogLevel))
	}
```

Reply:
Yeah happy to do that, was just following a pattern we had before.